content/platforms/pks/_index.md

```diff
@@ -0,0 +1,598 @@
+        The operator yaml file creates the operator deployment that is responsible for managing the k8s deployment and lifecycle of a Redis Enterprise Cluster.
+        Among many other responsibilities, it creates a [stateful set](https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/) that runs the Redis Enterprise nodes, as pods.
+        The yaml in the GitHub repository that you cloned contains the latest image tag representing the latest Operator version available.
+        This yaml does not require any changes under most circumstances.
+
+        To apply it:
+
+        ```src
+        kubectl apply -f operator.yaml
+        ```
+
+        You should receive the following response:
+
+        ```src
+        deployment.apps/redis-enterprise-operator created
+        ```
+
+1. Now, verify that your redis-enterprise-operator deployment is running:
+
+    ```src
+    kubectl get deployment -l name=redis-enterprise-operator
+    ```
+
+    A typical response will look like this:
+
+    ```src
+    NAME                        READY   UP-TO-DATE   AVAILABLE   AGE
+    redis-enterprise-operator   1/1     1            1           0m36s
+    ```
+
+1. Next, we'll create a storage class. The Redis Enterprise Cluster deployment dynamically provisions Persistent Volume Claims (PVCs) for use with cluster persistent storage needs.
+   In order to create dynamic PVCs, the Kubernetes cluster must have a storage class defined.
+   Determine whether a storage class is defined on your PKS cluster:
+
+    ```src
+    kubectl get storageclasses
+    ```
+
+    Since PKS does not automatically provision storage classes, and if you, or the cluster administrator, did not provision storage classes then the response will be:
+
+    ```src
+    No resources found.
+    ```
+
+    In order to create a storage class, determine the type of IaS your PKS cluster is running on, and consult the table in the [Kubernetes Storage Classes article](https://kubernetes.io/docs/concepts/storage/storage-classes/#provisioner) to determine which provisioner to use. Below, please find two examples of yaml files you can use for popular IaS providers:
+
+    - AWS - *gp2.yaml* <!-- Ben - I can't seem to make the code in the code blocks indent properly on Hugo. Please help, indentation is crucial here.  -->
+
+    ```src
+    apiVersion: storage.k8s.io/v1
+    kind: StorageClass
+    metadata:
+        name: gp2
+    mountOptions:
+        - debug
+    parameters:
+        type: gp2
+    provisioner: kubernetes.io/aws-ebs
+    reclaimPolicy: Retain
+    ```
+
+    - GCP - *standard.yaml*
+
+    ```src
+    apiVersion: storage.k8s.io/v1
+    kind: StorageClass
+    metadata:
+        name: standard
+    mountOptions:
+        - debug
+    parameters:
+        type: pd-standard
+    provisioner: kubernetes.io/gce-pd
+    reclaimPolicy: Retain
+    ```
+
+1. Create the appropriate yaml file to you IaS and apply it:
```

Comment:
This section needs some formatting improvement. I don't understand what is supposed to be commented.
You can see how it looks at: https://docs.redislabs.com/staging/pks-getting-started/platforms/pks/#step-2-get-and-prepare-deployment-files

Reply:
@amiramm I updated the indentation and it looks to me now to be the same as it was when you committed. Let me know if that's not correct.

Do you want uiServiceType and persistentSpec to be commented out?

Reply:
@bmansheim 
These are meant to be code blocks, contained within ```. They appear as plain text for some reason.
<img width="684" alt="Screen Shot 2019-07-30 at 9 44 18 PM" src="https://user-images.githubusercontent.com/7783341/62184555-73cbd700-b313-11e9-8955-00f425c7f1c5.png">

In this case, indentation is off.
<img width="701" alt="Screen Shot 2019-07-30 at 9 44 48 PM" src="https://user-images.githubusercontent.com/7783341/62184597-965df000-b313-11e9-9649-d547737d1de1.png">

Can you help with these, please?

Reply:
>Do you want uiServiceType and persistentSpec to be commented out?
no, those are fine now.